virttest/tests/unattended_install.py

```diff
@@ -406,6 +406,9 @@ def answer_kickstart(self, answer_path):
             else:
                 contents = dummy_graphical_re.sub('graphical', contents)
 
+        if self.params.get("cmd_only_use_disk"):
```

Comment:
could you give some notes or description for this cmd?

Reply:
Base on : https://github.com/autotest/tp-qemu/pull/1096
 serial_name = SYSTEM_DISK0
 blk_extra_params_image1 = "serial=${serial_name}"
cmd_only_use_disk = "ignoredisk --only-use=disk/by-id/*${serial_name}"

Description:
ignoredisk [--drives IGNOREDISK] [--only-use ONLYUSE] [--interactive]
Controls anaconda’s access to disks attached to the system. By default, all disks will be available for partitioning. Only one of the following three options may be used.

optional arguments:
--drives IGNOREDISK
    Specifies those disks that anaconda should not touch when partitioning, formatting, and clearing.

--only-use ONLYUSE
    Specifies the opposite - only disks listed here will be used during installation.

--interactive
    Allow the user manually navigate the advanced storage screen.